symplyphysics/laws/astronomy/luminosity_of_Sun_in_future_from_luminosity_in_present.py

```diff
@@ -0,0 +1,42 @@
+from sympy import (Eq, solve, S,)
```

Comment:
Use lower case in file name: `of_sun`

Reply:
Basically it is law about luminosity dependence on time. Therefore you can shorten name: `luminosity_of_sun_from_time`